net/mwan3/Makefile

```diff
@@ -21,7 +21,7 @@ define Package/mwan3
    SUBMENU:=Routing and Redirection
    DEPENDS:=+ip +ipset +iptables +iptables-mod-conntrack-extra +iptables-mod-ipopt
    TITLE:=Multiwan hotplug script with connection tracking support
-   MAINTAINER:=Jeroen Louwes <jeroen.louwes@gmail.com>
+   MAINTAINER:=Florian Eckert <fe@dev.tdt.de>
```

Comment:
@champtar on master i have updated this already. So i thought I should change this in this branch as well when i add some changes. @Adze1502 dropped the support for this package in master already. I agreed to maintenance this package for him.